### DISCUSSION TITLE
Pin a radio stream to the home screen?

### DISCUSSION THREAD

**Discussion by mirutoyoshi** [jcorporation/myMPD#499] [Q&A] [answered]
Is it possible to pin a favorite radio stream to the home screen?
I can add a playlist with the stream in it, but then i need to add the stream to the queue. Unless I'm missing something...

**Comment by jcorporation** [comment, marked as answer]
Add the playlist to the home screen and not the playlist view - Right click the playlist and select add to home screen.

**Reply by mirutoyoshi** [reply]
Yes that's worked, thanks!

**Reply by tsunulukai** [reply]
IMHO it would be great to be able to add individual songs / webradios on the home screen without having to define a playlist containing the stream url.
It's already feasible by manually editing the home_list state file and entering a extra "song" entry:

```
cat /var/lib/mympd/state/home_list
{"name":"La Première","ligature":"","bgcolor":"#ffffff","image":"radios_rtbf_1ere.png","cmd":"replaceQueue","options":["song","https://radios.rtbf.be/laprem1ere-128.mp3","Radio - La Première"]}
```

Would be great to have this natively integrated in the web UI.

**Reply by jcorporation** [reply]
This feature is added to v8.2.0
